pkgs/circuits/halo2-secp256k1-wasm/src/ecdsa.rs

```diff
@@ -16,11 +16,12 @@ use std::{cell::RefCell, rc::Rc};
 use wasm_bindgen::prelude::wasm_bindgen;
 
 #[derive(Debug)]
+// TODO: should it be private?
```

Comment:
if you need this struct in other module (other .rs file) it needs to have public visibility.